File: app/src/main/java/org/jellyfin/androidtv/browsing/CollectionActivity.java
package org.jellyfin.androidtv.browsing;

import android.os.Bundle;

import org.jellyfin.androidtv.base.BaseActivity;

public class CollectionActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getSupportFragmentManager()
                .beginTransaction()
                .replace(android.R.id.content, new CollectionFragment())
                .commit();
    }
}

File: app/src/main/java/org/jellyfin/androidtv/ui/browsing/CollectionActivity.kt
package org.jellyfin.androidtv.ui.browsing

import android.os.Bundle
import androidx.fragment.app.FragmentActivity

class CollectionActivity : FragmentActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		supportFragmentManager
			.beginTransaction()
			.replace(android.R.id.content, CollectionFragment())
			.commit()
	}
}
